contracts/misc/NFTManager.sol

```diff
@@ -0,0 +1,19 @@
+pragma solidity ^0.5.17;
+
+import "@openzeppelin/upgrades/contracts/Initializable.sol";
+import "@openzeppelin/contracts-ethereum-package/contracts/token/ERC721/IERC721.sol";
+import "@openzeppelin/contracts-ethereum-package/contracts/ownership/Ownable.sol";
+
+contract NFTManager is Initializable, Ownable {
+    function initialize() public initializer {
+        Ownable.initialize(msg.sender);
+    }
+
+    /// @notice Transfer specified NFT to a given recipient
+    /// @param recipient Address to transfer NFT to
+    /// @param nftContract NFT token contract to transfer from (e.g. Cryptokitties contract)
+    /// @param tokenId ID of specific NFT to transfer
+    function sendNFT(address recipient, IERC721 nftContract, uint tokenId) external onlyOwner {
+        nftContract.transferFrom(address(this), recipient, tokenId);
```

Comment:
should we use safeTransferFrom . see https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/token/ERC721/IERC721.sol#L59 .
might even better to use https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/token/ERC721/IERC721.sol#L128 as it gives more options to pass data along the call.

Reply:
My issue with safeTransferFrom() is the requirement that receiving contracts must implement IERC721Reciever - there may be a number of contract recipients that don’t implement this.

However, I agree safeTransfer should be the default option unless there’s a specific reason not to use it.

Here's my proposed change, with explicit functions for sending data or no data with safeTransfer(): https://github.com/dOrgTech/arc/pull/1/files

Reply:
please pr your proposal to this repo.